brave/brave-core
# Add missing icons to ftx widget

Resolves https://github.com/brave/brave-browser/issues/15928

<img width="348" alt="스크린샷 2021-06-15 오후 2 44 29" src="https://user-images.githubusercontent.com/6786187/121999487-58272800-cde8-11eb-98d8-0252297a0475.png">

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Connect ftx widget
2. Check icons in market tab
